cypress/integration/data_spec.js

```diff
@@ -1,31 +1,9 @@
-describe('Static Articles data', () => {
-  let responseStatus;
-  let responseBody;
-
-  const testScenarioResponse = (title, assertion) => {
-    it(title, () => {
-      assertion();
-    });
-  };
-
-  beforeEach(() => {
-    cy.request(`/data/test/scenario-01.json`).then(({ status, body }) => {
-      responseStatus = status;
-      responseBody = body;
-    });
-  });
+import { testResponseCode } from './test-helper';
 
-  testScenarioResponse('should return a 200 status code', () => {
-    expect(responseStatus).to.eq(200);
-  });
-
-  describe('Response Body', () => {
-    testScenarioResponse('should be an object', () => {
-      expect(responseBody).to.be.an('object');
-    });
-
-    testScenarioResponse('should contain a blocks object', () => {
-      expect(responseBody).to.have.property('blocks');
-    });
+describe('Static Articles data', () => {
+  describe('Page Status', () => {
+      it('should display 200', () => {
```

Comment:
Should clarify that this test is returning a 200 status code. `display 200` could mean a text of 200. The previous title of `should return a 200 status code` would suffice

Reply:
Thanks. I've made the change.